src/representations/feature_manager_dense.hh

```diff
@@ -159,6 +159,10 @@ namespace rascal {
                                 this->feature_size(), this->sample_size());
     }
 
+    inline Feature_Matrix_t get_feature_matrix_dense() {
+      return Feature_Matrix_t::Zero(2, 2);
```

Comment:
...Is this a placeholder? Otherwise, I don't understand what this is supposed to do.

Reply:
yes it is. I added a static_assert to avoid any call to it. In the near future the feature manager will disappear anyway.